Braintree.podspec

```diff
@@ -80,7 +80,7 @@ Pod::Spec.new do |s|
     s.source_files = "Sources/BraintreePayPalNativeCheckout/*.swift"
     s.dependency "Braintree/Core"
     s.dependency "Braintree/PayPal"
-    s.dependency "PayPalCheckout", '~> 0.100.0'
+    s.dependency "PayPalCheckout", '= 0.100.0'
```

Comment:
nit: Is this is the same as doing:
```
    s.dependency "PayPalCheckout", '0.100.0'
```
Is the `=` needed?

Reply:
Good question - the [Podspec](https://guides.cocoapods.org/syntax/podspec.html) docs don't call it out, but let me try locally with a `pod lib lint`

Reply:
Done!